### ISSUE TITLE
Provide options for DDC build (module format for example)

### ISSUE THREAD

**Issue by dam0vm3nt** [dart-lang/bazel#165]
It would be very usefull if in the `build.yaml` we could provide options for the `DDC` compiler like for example the module format to use.

**Comment by jakemac53** [comment]
agreed that would be good, although support needs to be added in the dart rules for other module formats first
